src/reducers/index.js

```diff
@@ -1,12 +1,10 @@
 import { combineReducers } from 'redux';
 import auth from './auth.reducer';
-
-// initial dummy state setup
-const app = (state = {intro: 'welcome to ah'}) => state
+import signInUser from './SignIn';
 
 const rootReducer = combineReducers({
-  app,
   auth,
+  signInUser,
```

Comment:
the naming convention for the reducer should be lowercase

Reply:
The signin reducer should be nested under the `auth` reducer like this.
```
auth: {
  signIn: {
    loading: false,
    error: ''
  },
  user: {},
  isLoggedIn: false
}
```
so the `signingIn` should be renamed to `loading` and the user object should be nested directly under the `auth` reducer